OpenUpSA/wazimap-ng-ui
# showing the selected attributes on point tooltip

## Description
Managing the visibility of the attributes on point tooltip

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/202

## How to test it locally
* on the point mapper select any category
* wait until the markers are created on the map
* hover on one of the markers to show the tooltip
* confirm that only the attributes that are contained by `visible_tooltip_attributes` property of the selected category are visible on the tooltip

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/114403542-8fd9df80-9bad-11eb-8c0c-7fd41d5bc88f.png)


## Changelog

### Added

### Updated
* `point_data.js` to show the selected attributes on the point tooltip

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
